tests/unittests/BackendTest.cpp

```diff
@@ -145,8 +145,8 @@ TEST_P(BackendTest, debugPrint) {
   IRBuilder(IR.get()).createDebugPrintInst("print", *IR->getWeights().begin());
 
   std::unique_ptr<Backend> backend(createBackend(GetParam()));
-  backend->init(std::move(IR));
-  backend->doForwardPass();
+  auto function = backend->compile(std::move(IR));
```

Comment:
nit: I prefer explicitly using * for auto pointers

Reply:
It's a `unique_ptr` though, so I don't think `auto *` is correct.